Makefile

```diff
@@ -0,0 +1,34 @@
+
+PHONY: test verify bootstrap go-bootstrap python-bootstrap
+
+build: 
+	godep go build .
+
+# test will run the python tests using the cli 
+test: build 
+	trial -j2 dvol_python \
+	&& HYPOTHESIS_PROFILE=ci trial dvol_python \
+	&& HYPOTHESIS_PROFILE=ci  DVOL_BINARY=$PWD/dvol trial -j2 dvol_python \
+	&& scripts/verify-tests.sh 
+
+# verify ensures your golang code passes 'the basics' 
+# locally before committing e.g. gofmt, go vet etc
+verify:
+	scripts/verify-preflight.sh 
+    
+bootstrap: go-bootstrap python-bootstrap
+
+# go-bootstrap installs all of the golang tools required by dvol
+# remember to add {GOPATH}/bin to your Path
+go-bootstrap:  
+	go get github.com/tools/godep \
+	&& go get golang.org/x/tools/cmd/cover \
+	&& go get golang.org/x/tools/cmd/vet \
+	&& go get golang.org/x/tools/cmd/goimports
```

Comment:
Suggested alternative factoring:

Rather than installing each of these in one big make target and then having `verify` call out to one monolithic shell script, we could instead have a target for each step that's currently in `verify-preflight.sh` and then have the `verify` target simply depend on each of those.

Then, as a separate thing, each of those targets (`gofmt`, etc) could depend on the necessary installers.

AFAICT the pros are that it puts everything in one place and makes dependencies more explicit and more granular, and the cons are pretty much the same, plus status qua bias.